src/components/Adventure.js

```diff
@@ -1,9 +1,171 @@
 import React from 'react';
+import $ from 'jquery';
+import { PropTypes } from 'prop-types';
+import '../styles/style.css';
+import Card from '@material-ui/core/Card';
+import Rating from '@material-ui/lab/Rating';
+import { makeStyles } from '@material-ui/core/styles';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import Fab from '@material-ui/core/Fab';
+
+const useStyles = makeStyles(() => ({
+  media: {
+    height: '284px',
+  },
+  card: {
+    padding: '20px 13px 32px',
+    height: '372px',
+  },
+  reviews: {
+    paddingLeft: '10px',
+    paddingRight: '10px',
+    paddingBottom: '0px',
+    paddingTop: '0px',
+    verticalAlign: 'top',
+    textAlign: 'center',
+  },
+  content: {
+    paddingLeft: '0px',
+    paddingTop: '0px',
+  },
+  title: {
+    marginBottom: '0px',
+    marginLeft: '5px',
+  },
+  button: {
+    marginLeft: 'auto',
+    marginRight: '0',
+    backgroundColor: 'black',
+    color: 'white',
+    textTransform: 'none',
+    borderRadius: '25px',
+    right: '-15%',
+  },
+
+  wrapper: {
+    position: 'relative',
+  },
+  infoi: {
+    position: 'absolute',
+    top: '2px',
+    right: '2px',
+    color: 'white',
+    background: 'transparent',
+    height: '24px',
+    width: '24px',
+  },
+  info: {
+    position: 'absolute',
+    top: '2px',
+    right: '2px',
+    color: '#FF6666',
+    background: 'transparent',
+    height: '24px',
+    width: '24px',
+  },
+  popular: {
+    background: '#E0E0E0',
+    fontWeight: 'bold',
+    radius: '50%',
+  },
+  bottom: {
+    padding: '8px 12px 12px',
+  },
+}));
+
+function Adventure({ adventure, updateLiked }) {
+  const classes = useStyles();
+
```

Comment:
can also destructure all of these at once, const { _id, name, image, reviews } = adventure